.github/workflows/test-application.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
                           SULU_ADMIN_EMAIL:
                           DATABASE_URL: "mysql://root:@127.0.0.1:3306/sulu_test?serverVersion=8.0"
 
-                    - php-version: '8.3'
+                    - php-version: '8.4'
```

Comment:
Scheb 2FA packages are not compatible with PHP 8.4 only the 7.0 so we can test against PHP 8.4 only on 2.6 branch.